spec/system/distribution_system_spec.rb

```diff
@@ -885,4 +885,27 @@
 
     expect(page).to have_content("This distribution has been marked as being completed!")
   end
+
+  context "When viewing the summary report" do
```

Comment:
Can we move these from system tests to request tests? System tests are slower, heavier and flakier, and there isn't any interaction in these test cases that would require them.

Reply:
done